TutorLizard.Web/appsettings.json

```diff
@@ -28,5 +28,11 @@
   },
   "ConnectionStrings": {
     "Default": ""
+  },
+  "Auth": {
+    "Google": {
+      "ClientId": "434224565561-jkpk0aeuvknvr9ffipo5cn2njq69077o.apps.googleusercontent.com",
```

Comment:
I don't think we should expose these in a public repository. You could have put a placeholder value here and sent us the actual values via Slack.